src/fides/config/cli_settings.py

```diff
@@ -45,8 +45,14 @@ class CLISettings(FidesSettings):
     @field_validator("server_url")
     @classmethod
     def get_server_url(cls, value: str, info: ValidationInfo) -> str:
-        """Create the server_url."""
+        """Create the server_url from the server configuration values. Strips path if present."""
         host = info.data.get("server_host")
+        # remove path if present from host
+        if "/" in host:
+            print(
+                f"server_host variable {host} has a forward slash, removing it. No path or porotocol should be present in the server_host variable."
+            )
+            host = host.split("/")[0]
```

Comment:
this will lead to some pretty unexpected results (which may be a bit hard to grok, for the end user) if the forward slash is part of a protocol prefix, or if it's some other unexpected input that isn't a path specification. i know the path specification is the primary use case we're trying to solve here, but maybe it'd be worth using [urllib.parse](https://docs.python.org/3/library/urllib.parse.html) to be a bit more precise/surefire in our handling? it's part of stdlib, so we wouldn't have to introduce a new dependency or anything.

let me know what you think!

Reply:
good point, here it does it with `urlparse` https://github.com/ethyca/fides/pull/5253/commits/579dd5b7f0610a5ad9e43816b8334a9c3d18ac9d#diff-5d19bcc00ba5a896a25504e3fd1154fa415dae45e11e9a4fa764e3a305916110R56